packages/lesswrong/components/ultraFeed/UltraFeedPostActions.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback } from "react";
+import DropdownMenu from "../dropdowns/DropdownMenu";
+import DropdownItem from "../dropdowns/DropdownItem";
+import { postGetPageUrl } from "@/lib/collections/posts/helpers";
+import SuggestCuratedDropdownItem from "../dropdowns/posts/SuggestCuratedDropdownItem";
+import { defineStyles, useStyles } from "../hooks/useStyles";
+import NotifyMeToggleDropdownItem from "../dropdowns/NotifyMeToggleDropdownItem";
+import { userGetDisplayName } from "@/lib/collections/users/helpers";
+
+const styles = defineStyles("UltraFeedPostActions", (theme: ThemeType) => ({
+  root: {
+    minWidth: 300
+  },
+}));
+
+const UltraFeedPostActions = ({ post, closeMenu, includeBookmark }: {
+  post: PostsListWithVotes,
+  closeMenu: () => void,
+  includeBookmark?: boolean,
+}) => {
+  const classes = useStyles(styles);
+
+  const handleOpenNewTab = useCallback((ev: React.MouseEvent) => {
+    ev.preventDefault();
+    window.open(postGetPageUrl(post), "_blank", "noopener,noreferrer");
+    closeMenu();
+  }, [post, closeMenu]);
+
+  const handleCopyLink = useCallback(() => {
+    const url = postGetPageUrl(post, true);
+    void navigator.clipboard.writeText(url);
+    closeMenu();
+  }, [post, closeMenu]);
+
+  const author = post.user;
```

Comment:
You don't want the same conditional on it not being the current user for displaying the "notify me" item?